### ISSUE TITLE
Bash not accepting any input after "some tome"

### ISSUE THREAD

**Issue by thenitai** [Microsoft/BashOnWindows#1741]
This is happening new now with 15046.

Work in Bash, step away for like 30 minutes, Bash this there and all but it doesn't accept any keystrokes anymore. Nothing, nada, zip. Only remedy is to close the window. Opening another bash window works.

**Comment by sunilmut** [comment]
@thenitai - Interesting! Thanks for the post. I have not been able to repro this behavior on 15046 build, even after keeping bash idle for hours. Are you seeing this consistently on this build?

Curious to know if anyone else is hitting this.

**Comment by thenitai** [comment]
@sunilmut Yep, just happened again. Left my machine running. came back and boom... Bash doesn't accept any inputs. All other apps are fine

**Comment by therealkenc** [comment]
Possibly #2519 (or related) which remains open. It is possible Sunil can't repro because his machine doesn't sleep.
